openfisca_france_local/parameters/departements/eure_et_loir/repas_foyer.yml

```
@@ -0,0 +1,37 @@
+repas_foyer:
+  age_minimal_personne_agee_apte_travail:
+    reference: Titre 2 Chapitre 1-2 Article 36 du Règlement départemental d'Aide Sociale PA PH de l'Eure et Loir
```

Typo : 
```suggestion
    reference: Titre 2 Chapitre 1-2 Article 36 du Règlement départemental d'Aide Sociale PA PH de l'Eure-et-Loir
```
De même pour les autres paramètres.